changelog.d/pa-3298.fixed

```diff
@@ -0,0 +1,10 @@
+taint-mode: Semgrep will now treat lambdas' parameters as fresh, so a taint rule
+that to find double-delete's should no trigger on the code below:
```

Comment:
nit:
so a taint rule that to find double-delete's should no trigger on ->
so a taint rule that finds double-delete's should not be triggered on

Reply:
Thanks for the correction!